enterprise/server/util/pebble/pebble.go

```diff
@@ -375,6 +379,20 @@ func (idb *instrumentedDB) NewIndexedBatch() Batch {
 	return &instrumentedBatch{batch, batch, idb}
 }
 
+func Peek(dirname string, fs FS) (*DBDesc, error) {
```

Comment:
I don't think we get much out of instrumenting this? I think a simple `var Peek = pebble.Peek` would be sufficient.

Reply:
Sure, I don't feel too strongly either way, removed it.